devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy

```diff
@@ -34,6 +34,9 @@ metadata {
 		fingerprint profileId: "0104",  inClusters: "0000, 0001, 0003, 0009, 0402", manufacturer: "HEIMAN", model: "HT-EM", deviceJoinName: "HEIMAN Multipurpose Sensor" //HEIMAN Temperature & Humidity Sensor
 		fingerprint profileId: "0104",  inClusters: "0000, 0001, 0003, 0402, 0B05", manufacturer: "HEIMAN", model: "HT-EF-3.0", deviceJoinName: "HEIMAN Multipurpose Sensor" //HEIMAN Temperature & Humidity Sensor
 		fingerprint profileId: "0104", deviceId: "0302", inClusters: "0000,0001,0003,0020,0402,0405", outClusters: "0003,000A,0019", manufacturer: "frient A/S", model :"HMSZB-110", deviceJoinName: "frient Multipurpose Sensor" // frient Humidity Sensor
+
+		//eWeLink
+		fingerprint profileId: "0104", inClusters: "0000, 0001, 0003, 0402, 0405",  outClusters: "0003", manufacturer: "eWeLink", model: "TH01", deviceJoinName: "eWeLink Temp/Humidity Sensor"
```

Comment:
change `deviceJoinName` to `"eWeLink Multipurpose Sensor"`

Reply:
I have modified the file.